cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -1011,6 +1011,10 @@ declare namespace Deno {
     /** Restarts the timer for the bench measurement. This should be called
      * after doing setup work which should not be measured.
      *
+     * Warning: This method should not be used for benchmarks averaging less
+     * than 0.01s per iteration. In such cases it will be disabled but the call
+     * will still have meaningless overhead, and so a warning will be printed.
```

Comment:
```suggestion
     * Warning: This method should not be used for benchmarks averaging less
     * than 0.01s per iteration. In such cases it will be disabled but the call
     * will still have noticeable overhead, resulting in a warning.
```